sdk/storage/storage-file-share/test/node/highlevel.node.spec.ts

```diff
@@ -404,7 +404,7 @@ describe("Highlevel Node.js only", () => {
     assert.ok(downloadedData.equals(uploadedData));
   });
 
-  it("fileClient.download should download partial data when internal stream unexcepted ends", async () => {
+  it("fileClient.download should download partial data when internal stream unexpected ends", async () => {
```

Comment:
Test titles have been changed, which is why the recordings aren't found, update the names of the corresponding recording files to get everything working. No need to re-record.